lib/net.js

```diff
@@ -367,9 +367,16 @@ Socket.prototype.address = function() {
 };
 
 
+Object.defineProperty(Socket.prototype, '_connecting', {
+  get: function() {
```

Comment:
@evanlucas it should be, will do it in a follow-up PR, because it won't be backported.